Sources/FndBase/Utils/Utils.swift

```diff
@@ -0,0 +1,13 @@
+import CommonSources
+
+public func fetchLocalConfig(network: TorusNetwork) throws -> AllNodeDetailsModel? {
+    
+    switch network {
+        case .legacy(let network ) :
```

Comment:
plz refer to this function here: https://github.com/torusresearch/fetch-node-details/blob/d76f9f382cf22874ba97a6933e28756169b2867f/packages/fnd-base/src/utils.ts#L14C17-L14C33

Reply:
it uses migration map to determine correct config